BookingsAPI/Bookings.DAL/Mappings/ParticipantMap.cs

```diff
@@ -16,12 +16,15 @@ public void Configure(EntityTypeBuilder<Participant> builder)
             builder.Property(x => x.DisplayName);
             builder.Property(x => x.CaseRoleId);
             builder.Property(x => x.HearingRoleId);
+            builder.Property(x => x.QuestionnaireId);
 
             builder.HasOne(x => x.CaseRole).WithMany().HasForeignKey(x => x.CaseRoleId);
             builder.HasOne(x => x.HearingRole).WithMany().HasForeignKey(x => x.HearingRoleId);
+            builder.HasOne(x => x.Questionnaire).WithMany().HasForeignKey(x => x.QuestionnaireId);
 
             builder.HasOne<Hearing>("Hearing").WithMany("Participants").HasForeignKey(x => x.HearingId);
-            builder.HasMany<SuitabilityAnswer>("SuitabilityAnswers").WithOne("Participant").HasForeignKey(x => x.ParticipantId);
+
+           // builder.HasMany<SuitabilityAnswer>("SuitabilityAnswers").WithOne("Participant").HasForeignKey(x => x.ParticipantId);
```

Comment:
Is this a work in progress or deleted code? Using this would probably be what would make the deletes cascade.

Reply:
deleted